bukuserver/views.py

```diff
@@ -138,6 +142,25 @@ def _list_entry(self, context: Any, model: Namespace, name: str) -> Markup:
             res += [description]
         return Markup("".join(res))
 
+    @pass_context
+    def get_detail_value(self, context, model, name):
+        value = super().get_detail_value(context, model, name)
+        if name == 'tags':
+            tags = (link(s.strip(), url_for('bookmark.index_view', flt0_tags_contain=s.strip()), badge='default')
+                    for s in (value or '').split(',') if s.strip())
+            return Markup(f'<div class="tag-list">{"".join(tags)}</div>')
+        if name == 'url':
+            res, netloc, scheme = [], (parsed := urlparse(value)).netloc, parsed.scheme
+            if netloc and not app_param('DISABLE_FAVICON', False):
+                icon = f'<img class="favicon" title="netloc:{netloc}" src="http://www.google.com/s2/favicons?domain={netloc}"/>'
+                res += [link(icon, url_for('bookmark.index_view', flt0_url_netloc_match=netloc), html=True)]
+            elif netloc:
+                badge = f'<span class="netloc">netloc:{escape(netloc)}</span>'
+                res += [link(badge, url_for('bookmark.index_view', flt0_url_netloc_match=netloc), html=True, badge='success')]
+            res += [escape(value) if not scheme else link(value, value, new_tab=app_param('OPEN_IN_NEW_TAB'))]
```

Comment:
This reproduces current logic for rendering URL as a hyperlink within `_list_entry()`